src/blockchain/download-augur-logs.ts

```diff
@@ -69,7 +70,9 @@ async function processBatchOfLogs(db: Knex, augur: Augur, allAugurLogs: Array<Fo
       await each(logs, async (log) => await insertTransactionHash(trx, blockNumber, log.transactionHash));
       logger.info(`Processing ${dbWriteFunctions.length} logs`);
       for (const dbWriteFunction of dbWriteFunctions) {
-        await dbWriteFunction(trx);
+        await dbWriteFunction(trx).catch((err) => {
```

Comment:
Did you mean to leave this in? This will swallow the error and proceed

Reply:
Oops yeah this should be removed. Thanks!